.travis.yml

```diff
@@ -55,7 +55,9 @@ install:
 
 before_script:
   - if [[ "$TRAVIS_OS_NAME" == "osx" && "$TRAVIS_PULL_REQUEST" == false ]]; then ./ci-scripts/add-key.sh; fi
-  - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then export DISPLAY=:99.0 && sh -e /etc/init.d/xvfb start; fi
+
+services:
```

Comment:
Why is this changed?

Reply:
that's how it works now in xenial. the old `before_script` execution gives error

Reply:
https://changelog.travis-ci.com/ubuntu-xenial-16-04-build-environment-update-86123

Reply:
The old way doesn't run on OS X, the new way does?   You link doesn't mention xvfb